### ISSUE TITLE
Automatically process pixel perfect duplicates

### ISSUE THREAD

**Issue by Gradivus500** [hydrusnetwork/hydrus#1197]
When ever I do duplicate processing and run across a pixel perfect dupe, I always just pick the one with the smaller file size. It would be nice if there was a button that just did this for my entire DB, so I don't waste time manually sorting through them.

**Comment by Chad90b** [comment]
this is being considered/worked on. a few bugs have existed that needed to be addressed before something like this could ever exist for example #1158 plus a lot of users have different things they want out of something like this, for example certain rules, so it might take a while for any version of this to be released. but you have been heard.

your request is related to #297 so im tagging it here

**Comment by Gradivus500** [comment]
Yeah I saw 1158  and meant to link it here as a prerequisite to this being implemented. This seems to be a more specific/narrow view application of what is mentioned in 297, so I can close this if you think that constitutes a duplicate.
